content/source/docs/enterprise/admin/logging.html.md

```diff
@@ -1,86 +1,303 @@
+```
+tfe-admin app-config -k log_forwarding_config -v "$(cat fluent-bit.conf)"
+```
 
-  * Authentication Tokens
-  * Configuration Versions
-  * Policy Versions
-  * OAuth Tokens
-  * SSH Keys
-  * State Versions
-  * Users
-  * Variables
+Once configured, the Terraform Enterprise application settings should show the
+`log_forwarding_config` setting in escaped JSON string format:
 
-When requests occur, these pieces of information will be logged:
+```
+    ...
+    "log_forwarding_config": {
+        "value": "# Match all logs and do not forward them anywhere.\n[OUTPUT]\n    Name null\n    Match *\n"
+    },
+    ...
+```
 
-  1. The actor
-    * Users (including IP address)
-    * Version Control System users (identified in webhooks)
-    * Service accounts
-    * Terraform Enterprise
-  2. The action
-    * Reading sensitive resources
-    * Creation of new resources
-    * Updating existing resources
-    * Deletion of existing resources
-    * Additional actions as defined in /actions/* namespaces
-    * Webhook API calls
-  3. The target of the action (any resource exposed by the V2 API)
-  4. The time that the action occurred
-  5. Where the action was taken (web/API request, background job, etc.)
+The default configuration does not forward any logs.
 
-### Log Format
+```
+# Match all logs and do not forward them anywhere.
+[OUTPUT]
+    Name null
+    Match *
+```
+
+To forward logs to multiple external destinations, use multiple `[OUTPUT]`
+directives.
+
+```
+# Forward all logs to Datadog.
+[OUTPUT]
+    Name datadog
+    Match *
+    ...
+
+# Forward all logs to Fluent Bit or Fluentd.
+[OUTPUT]
+    Name forward
+    Match *
+    ...
+```
+
+~> **NOTE:** Do not use an `[OUTPUT]` directive with the
+[`stdout` Fluent Bit output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/standard-output).
+Doing so will result in a loop that will continuously emit logs!
+
+## Supported External Destinations
+
+These are the supported external destinations one can forward logs to. Each
+supported external destination contains example configuration for convenience.
+
+### Amazon CloudWatch
 
-Log entries are in JSON, just like other Terraform Enterprise logs. Most audit log entries are formatted like this:
+This example configuration forwards all logs to Amazon CloudWatch. Refer to the
+[`cloudwatch_logs` Fluent Bit output plugin documentation](https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch)
+for more information.
 
-``` json
-{
-  "timestamp": "2017-12-19T15:23:45.148Z",
-  "resource": "workspace",
-  "action": "destroy",
-  "resource_id": "ws-9a3hrbYfFsTzg2FZ",
-  "actor": "jsmith",
-  "actor_ip": "94.122.17.37"
-}
 ```
+[OUTPUT]
+    Name               cloudwatch_logs
+    Match              *
+    region             us-east-1
+    log_group_name     example-log-group
+    log_stream_name    example-log-stream
+    auto_create_group  On
+```
+
+~> **NOTE:** Sending to Amazon CloudWatch is only supported when Terraform
```

Comment:
We've got a lot of ~Notes~ going on, perhaps we move this (and the gcp one) just to the top of the appropriate section instead?

Reply:
I moved this information to the top of its respective section and removed the note formatting for it. Let me know if that reads better.